Kitodo/src/main/resources/messages/errors_en.properties

```diff
@@ -25,7 +25,7 @@ calendar.upload.isEmpty=No file was transferred.
 calendar.upload.missingMandatoryElement=A mandatory XML element could not be found.
 calendar.upload.missingMandatoryValue=A mandatory value could not be found.
 calendar.upload.overlappingDateRanges=Due to overlapping date ranges of blocks, the course of appearance could not be imported\:
-catalogError=Error querying OPAC \u201E{0}\u201C
+catalogError=Error querying OPAC „{0}“
```

Comment:
```suggestion
catalogError=Error querying OPAC "{0}"
```

Reply:
You used the correct replacement, but I think that for this English text the old characters were wrong, which is fixed by my suggestion.

Reply:
@solth, this one is still pending.